buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1646,8 +1656,10 @@ class BeamModulePlugin implements Plugin<Project> {
       project.ext.applyJavaNature(
           exportJavadoc: false,
           enableSpotbugs: false,
-          enableChecker: false,
           publish: configuration.publish,
+          generatedClassPatterns: [
+            '^org\\.apache\\.beam\\..*'
+          ],
```

Comment:
This looked concerning until I looked around and realized the "portability" nature means its for making modules that generate protobuf/grpc code. The comment above "applyPortabilityNature should only be applied to projects that want to use vendored gRPC / protobuf dependencies" isn't very helpful. Maybe I'll send you a PR to document these natures.

Reply:
Yea. I could be `.model` except it is also used by the windmill proto generation. I have changed it so that it is passed in and the default is the model.